### DISCUSSION TITLE
Custom wide themes (v2 only)

### DISCUSSION THREAD

**Discussion by AndreasK79** [magicbug/Cloudlog#1421] [Show and tell]
As a wide screen user, I took the Superhero theme and modified it to be wider, and to give more space to the map.
You need V2, which is under development, to add a custom theme.

For those interested, you can find the theme here: https://github.com/AndreasK79/cloudlog_wide_custom_themes

Previews:
![image](https://user-images.githubusercontent.com/6977712/157098662-5ed184b4-511f-44b6-9c26-266fb44f61aa.png)
![image](https://user-images.githubusercontent.com/6977712/157098824-1b1ebd93-e4f3-4855-b555-197d5e22c8dc.png)

Just an update that I now have added wide versions of all themes included in Cloudlog.

**Comment by phl0** [comment]
Just installed and looks very nice. I like the wider screen layout. Maybe also an option to adapt similar changes to the dark Cyborg theme :)

**Reply by AndreasK79** [reply]
I might give it a go at some point. Anyway, this is the reason I added custom themes to V2, and I changed so that only 2 files are needed for a theme.

If I remember correctly, most (if not all) of the changes are contained in overrides.css.

**Reply by phl0** [reply]
I'll try to diff your changes against the original superhero theme and then make similar to Cyborg ;-)

**Reply by AndreasK79** [reply]
Yeah, that would be my approach as well. I'll happily include a folder with a wider cyborg theme, if you submit a PR.

**Reply by phl0** [reply]
Sounds like a good deal :)

**Reply by AndreasK79** [reply]
@phl0 I took a look. All the modifications can be placed in overrides.css. The only difference that needs a modification between themes, are the colors of the scrollbar.

I can submit another commit with a wide Cyborg theme now if you want.

**Reply by AndreasK79** [reply]
@phl0 I added wide versions for all themes

**Reply by phl0** [reply]
cnfm. Installed and tested. Looks good to me. Only issue that costed my ~15 mins is that this repo has a typo in its name: "[cloudloug_wide_custom_themes](https://github.com/AndreasK79/cloudloug_wide_custom_themes)". There is one "u" too much ;-)

**Reply by AndreasK79** [reply]
@phl0 fat fingers. Thanks for pointing it out. I've fixed it now.
